README.md

```diff
@@ -126,8 +126,12 @@ involved first. You can ping others in the USRSE slack to contribute, and it's a
 Document for easier commenting and editing, and then move to the wiki when it's in a final first draft state. 
 For all community documents, the following points apply:
 
-1. It's strongly suggested to first work on your document in Google Docs or similar, where you can bring in multiple community members to put together a first draft. Importantly, you should make sure that the content adheres to the US-RSE [Code of Conduct](https://us-rse.org/code-of-conduct/). At least 3 community members (including yourself) are required before submitting the document to the site. You should feel that it's in a solid "first draft state" before doing a submission.
-2. You should verify that 1) the content is of interest to the RSE community, and 2) the content does not violate the Code of Conduct. Any material added to the wiki that does not follow these rules is subject to removal. If you see content on the wiki that is not appropriate or otherwise breaks these rules, please open an issue on the repository immediately, or contact the steering committee.
+1. It's strongly suggested to first work on your document in Google Docs or similar, where you can bring in multiple community members to put together a first draft. Importantly, you should make sure that the content adheres to the US-RSE [Code of Conduct](https://us-rse.org/code-of-conduct/). At least three community members (including yourself) are required before submitting the document to the site. You should feel that it's in a solid "first draft state" before doing a submission.
+2. Make sure that you verify the following:
+  - the content is of interest to the RSE community
```

Comment:
@vsoch One small change is needed. If you see the preview of this README, the bullets are not aligned properly.
<img width="400" alt="Screenshot 2021-03-19 at 12 11 54 PM" src="https://user-images.githubusercontent.com/1132451/111741353-65e21580-88ac-11eb-9896-b4563eb243a9.png">

Use tab instead of space before `-`.

Reply:
I'm okay with how it looks, but if you want to suggest a change I'd be happy to accept it.